tutorials/early_stopping/early_stopping.ipynb

```diff
@@ -0,0 +1,978 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "8158a8f5",
+   "metadata": {},
+   "source": [
+    "## Early stopping in Ax\n",
+    "\n",
+    "This tutorial illustrates how to add an early stopping strategy to an Ax HPO loop. Most of this tutorial is adapted from the [PyTorch Ax Multiobjective NAS Tutorial](https://pytorch.org/tutorials/intermediate/ax_multiobjective_nas_tutorial.html). The training job is different from the original in that we do not optimize `batch_size` or `epochs`. This was done for illustrative purposes, as each validation curve now has the same number of points. The companion training file from the NAS tutorial `mnist_train_nas.py` has also been altered to log to Tensorboard during training.\n",
```

Comment:
Let's talks about "trial-level early stopping" to be precise and distinguish this from experiment-level early stopping. Maybe it's good to add a high-level discussion how trial-level ES works and why one may want to use it?